src/inventory/ModelManager.h

```diff
@@ -107,6 +113,14 @@ struct Model : ModelMeta {
         return QString("%1%2").arg(shortName).arg(qHash(repositoryUrl));
     }
 
+    inline QString getReportedRepo() const {
+        // Models could come with their own repository specified in model_info.json
+        // However it is more reliable to remember where one model came from, by setting the
+        // repository URL when downloading. That being sad, sometimes it is useful to specify this
+        // for builtin models so that we remember that this is a self managed model
+        return repo;
+    }
+
```

Comment:
Why does this one get a getter? Because you want to be able to specify a default value for local models that don't have the info in model_info.json?

Reply:
Aah, okay, makes sense!